lib/ohai/plugins/linux/virtualization.rb

```diff
@@ -188,6 +216,18 @@ def nova_exists?
         # But, this could overwrite virtualization[:systems][:lxc] = "guest"
         # If so, we may need to look further for a differentiator (OHAI-573)
         virtualization[:systems][:lxc] = "host"
+      elsif init_environ_content =~ %r{\x00?container=lxc\x00?}
```

Comment:
@stefanor Possibly yes, but i've never seen this parameter at the beginning or the end of /proc/1/environ